include/libpldm/oem/ibm/state_set.h

```diff
@@ -16,6 +16,7 @@ enum ibm_oem_pldm_state_set_ids {
 	PLDM_OEM_IBM_SBE_SEMANTIC_ID = 32775,
 	PLDM_OEM_IBM_SBE_HRESET_STATE = 32776,
 	PLDM_OEM_IBM_BOOT_SIDE_RENAME = 32773,
+	PLDM_OEM_IBM_SBE_DUMP_UPDATE_STATE = 32777,
```

Comment:
just did a quick grep and figured that we also have 32777 stateset id with PHYP, that is used for panel ..its wierd that we picked it that way ..but i think we also use the tid , entity id in combination .. probably repeating the same might not harm much ..